File: src/main/java/com/alxnns1/mobhunter/entities/neopterans/renderer/HornetaurRenderer.java
package com.alxnns1.mobhunter.entities.neopterans.renderer;

import com.alxnns1.mobhunter.MobHunter;
import com.alxnns1.mobhunter.entities.neopterans.HornetaurEntity;
import com.alxnns1.mobhunter.entities.neopterans.model.HornetaurModel;
import net.minecraft.client.renderer.entity.EntityRendererManager;
import net.minecraft.client.renderer.entity.MobRenderer;
import net.minecraft.util.ResourceLocation;

import javax.annotation.Nonnull;

public class HornetaurRenderer extends MobRenderer<HornetaurEntity, HornetaurModel<HornetaurEntity>> {
	public HornetaurRenderer(EntityRendererManager renderManagerIn) {
		super(renderManagerIn, new HornetaurModel<>(), 0.5f);
	}

	@Nonnull
	@Override
	public ResourceLocation getEntityTexture(@Nonnull HornetaurEntity entity) {
		return new ResourceLocation(MobHunter.MOD_ID, "textures/entity/hornetaur.png");
	}
}

File: src/main/kotlin/com/alxnns1/mobhunter/entity/neopteran/renderer/HornetaurRenderer.kt
package com.alxnns1.mobhunter.entity.neopteran.renderer

import com.alxnns1.mobhunter.MobHunter
import com.alxnns1.mobhunter.entity.MHRenderer
import com.alxnns1.mobhunter.entity.neopteran.HornetaurEntity
import com.alxnns1.mobhunter.entity.neopteran.model.HornetaurModel
import net.minecraft.client.renderer.entity.EntityRendererManager
import net.minecraft.util.ResourceLocation

private val RL = ResourceLocation(MobHunter.MOD_ID, "textures/entity/hornetaur.png")

class HornetaurRenderer(rendererManager: EntityRendererManager) : MHRenderer<HornetaurEntity, HornetaurModel>(rendererManager, HornetaurModel(), 0.5F) {

	override fun getEntityTexture(entity: HornetaurEntity): ResourceLocation = RL
}
